__tests__/endpoints.test.js

```diff
@@ -0,0 +1,53 @@
+const request = require("supertest");
+const app = require("../app");
+const seed = require("../db/seeds/seed");
+const data = require("../db/data/test-data/index");
+const db = require("../db/connection");
+
+
+
+beforeEach(() => {
+    return seed(data);
+  });
+  
+  afterAll(() => {
+    return db.end();
+  });
+  
+  
+  describe("api/topics", () => {
+      test("status check responds 200", () => {
```

Comment:
We should not just test the status because we are checking it in the next test as well, feel free to remove this test block 🙂